posthog/api/cohort.py

```diff
@@ -308,6 +308,10 @@ def safely_get_queryset(self, queryset) -> QuerySet:
         if self.action == "list":
             queryset = queryset.filter(deleted=False)
 
+            search_query = self.request.query_params.get("search", None)
+            if search_query:
+                queryset = queryset.filter(name__icontains=search_query)
```

Comment:
added search functionality to this endpoint using the built-in django ORM search.